### PR TITLE
upd(brave): switch maintainers and update packages

### PR DESCRIPTION
**Pull request by Henryws**
I've decided to start maintaining Brave, as the previous maintainers have stepped down from doing so.

@WRM-42 do you want to continue maintaining certain Brave Pacscripts?

### CODE REVIEW

**Comment by WRM-42** [comment]
I don't have any plans to stop maintaining them any time soon. I will try to organize myself to find more free time, as these pacscripts are quickly becoming outdated.

**Comment by wizard-28** [comment]
> I don't have any plans to stop maintaining them any time soon. I will try to organize myself to find more free time, as these pacscripts are quickly becoming outdated.

Well, we're brewing a tool to make maintainers' lives easier. It's not done yet, but the repository is https://github.com/pacstall/pacup. You can join our discord/matrix if you want to participate in it's development before I push the commits to github.

**Comment by wizard-28** [comment]
> I've decided to start maintaining Brave, as the previous maintainers have stepped down from doing so.
> 
> @WRM-42 do you want to continue maintaining certain Brave Pacscripts?

Tagging @Zahrun as he maintains some of those pacscripts.

**Comment by Zahrun** [comment]
I’m not planning to maintain actively the brave-deb pacscript.
I find it confusing that we cannot subscribe to only the stable releases for brave.
At least I didn’t find a way.
Thank you

**Review by wizard-28 — CHANGES_REQUESTED** [review]
@Henryws do the repology thingy

File: packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 gives="brave-browser-nightly"
-breaks="${gives}-bin ${gives}-app"
+breaks="${gives}-bin ${gives}-app ${gives}-git"
+replace="${gives}"
 url="https://github.com/brave/brave-browser/releases/download/v${version}/${gives}_${version}_amd64.deb"
 description="The Brave browser is a fast, private and secure web browser for PC, Mac and mobile. This is the nightly build of Brave Browser"
-hash="4b656207fab91977a59a529a5ff2502c5e9f5969838174f11c4dbfeff46e37b8"
-maintainer="Korkmatik <9c54efc1-143b-4f5f-9d2f-f6f189583f0f@aleeas.com>"
+hash="e0a9405379041a8557d80855dd7786ed6fdbb08c9da98b736e11fd0234983afc"
+repology=("project: brave" "visiblename: brave-nightly-bin")
```
Comment (wizard-28) [inline packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript:9]: You might want to Scoop for nightly, they have the latest one

img

```suggestion
repology=("project: brave" "visiblename: brave-nightly")
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Check for updates in all of the pacscripts here.

File: packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript (hunk 1 above)
Comment (wizard-28) [reply packages/brave-browser-nightly-deb/brave-browser-nightly-deb.pacscript:9]: @Henryws your thoughts?

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Update all of them. They're outdated.

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**